cmd/forch

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash -e
 
-SOCK=inst/faucet_event.sock
+echo Script name: "$0"
+
+if [ -z "$1" ]
+then 
+    SOCK=inst/faucet_event.sock
+else
+    SOCK=$1
```

Comment:
shouldn't this be inst/faucet/daq-faucet-$1/faucet_event.sock?  Use should match cmd/faucet

Reply:
I am just passing the full file socket path to it. Should we leave it generic as it is or do I add the daq-faucet-$1 location.

Reply:
Yes, it should match cmd/faucet -- so "cmd/faucet elephant" would be compatible with "cmd/forch elephant"